### PR TITLE
format the stats depending on the language

### PR DESCRIPTION
**Pull request by MilanBarande**
[Jira Ticket](https://bluenove.atlassian.net/browse/DEVAS-1504)

I just created a global function to format the number following the user story and use it in headerStatistics

### CODE REVIEW

File: assembl/static2/js/app/utils/globalFunctions.js [hunk 1]
```diff
@@ -17,6 +17,11 @@ export const getDiscussionId = () => getInputValue('discussion-id');
 
 export const getDiscussionSlug = () => getInputValue('discussion-slug');
 
+export const formatNumber = (number: number | string, locale: string) => {
+  const separator = locale !== 'en' ? ' ' : ',';
+  return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, separator);
+};
+
```
Comment (vincentfretin) [inline assembl/static2/js/app/utils/globalFunctions.js]: You are only supporting two languages here. Please use existing lib for this. Look at react-redux-i18n Localize component for this which uses the standard Intl.NumberFormat behind the scene.

**Comment by vincentfretin** [comment]
https://www.npmjs.com/package/react-redux-i18n
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat

File: assembl/static2/js/app/utils/globalFunctions.js (hunk 1 above)
Comment (MilanBarande) [reply assembl/static2/js/app/utils/globalFunctions.js]: Well the story only talks about dealing with english and every other language should have whitespaces. But I agree if we have a simple way to deal with every language why not. Need to take a look at this, I didn't know about it

File: assembl/static2/js/app/components/common/headerStatistics.jsx [hunk 2]
```diff
@@ -10,52 +10,54 @@ type StatisticElementProps = {
 
 type StatElements = Array<StatisticElementProps>;
 
-export const statContributions = (numContributions: number): StatisticElementProps => ({
+export const statContributions = (numContributions: number) => ({
```
Comment (vincentfretin) [inline assembl/static2/js/app/components/common/headerStatistics.jsx]: Why did you remove `: StatisticElementProps`?

File: assembl/static2/js/app/components/common/headerStatistics.jsx [hunk 3]
```diff
@@ -40,22 +40,24 @@ export const statParticipations = (numParticipations: number): StatisticElementP
...
+          <Translate value={metricNameTranslateKey} count={metricValue} />
         </div>
       </div>
     </div>
   </div>
 );
 
 const mapElementsPropsToComponents = elemsProps =>
-  elemsProps.map((elementProps, index) => <StatisticElement key={index} {...elementProps} />);
+  elemsProps.map((elementProps, index) => <StatisticElement key={`stat-${index}`} {...elementProps} />);
```
Comment (vincentfretin) [inline assembl/static2/js/app/components/common/headerStatistics.jsx]: you can use elemsProps.metricNameTranslateKey as key instead to avoid the eslint warning
